build-setup-nolog.sh

```diff
@@ -143,6 +143,10 @@ if wget -T 1 -t 3 -O /dev/null http://169.254.169.254/; then
     sudo pip3 install -r sw/firesim-software/python-requirements.txt
     cat sw/firesim-software/centos-requirements.txt | sudo xargs yum install -y
 
+    # Setup for using qcow2 images
+    cd $RDIR
+    ./scripts/install-nbd-kmod.sh
```

Comment:
can this be put in machine launch?

Reply:
I wanted to put this here for two reasons:
1. It could be brittle to start out, so it's easier to get a log for it this way/fix it
2. If something updates the kernel version, it needs to be rebuilt